meta/main.yml

```diff
@@ -11,5 +11,5 @@ galaxy_info:
     - trusty
   categories:
   - development
-dependencies:
-  - { role: "azavea.java" }
+#dependencies:
```

Comment:
Indeed, we don't want to hardcode such a base dependency .

ps: I'm still waiting for any fix to: 
> Unfort. ansible doesn't yet offer a feature to easily disable 'transient dependencies'.
I created an ansible feature request: ansible/ansible#13215 , but will probably take time to get it builtin (hopefully for v2.1)